### ISSUE TITLE
What will happen to this library?

### ISSUE THREAD

**Issue by jmorrn** [blackuy/react-native-twilio-video-webrtc#738]
Recently Twilio announced that they are going to stop maintaning the Twilio Video product, right after it, the following questions came to my mind:

- What will happen to 'react-native-twilio-video-webrtc'?
- Is there any plans of stoping the maintenance?
- If any part of the Twilio Video stops working before December 5, will it be fixed?

**Twilio Doc in which they announced it**

https://www.twilio.com/docs/video

**Comment by gastonmorixe** [comment]
Yeah, definitely needs some love and support. We hope to bring you some updates and answers on all those questions ASAP.
Thank you for asking, just pinned this issue.

**Comment by gastonmorixe** [comment]
Good news, Twilio just announced that Twilio Video will remain a core service. Please read their announcement here: https://www.twilio.com/en-us/blog/twilio-video-update-2024

About this library, we are still in talks with them to find a way to improve it and add many features. We'll keep you posted. 

We are also open to companies that may want to sponsor the development of it, feel free to reach out. My contact is in my profile.

Thank you all for your patience!

**Comment by flexbox** [comment]
Hello everyone

> Twilio Video stops working before December 5, will it be fixed?

Does it means the status is "maintenance mode" or do we will have new features to keep up with react native ecosystem —_like new architecture support or last Expo SDK fixes see #758_

**Comment by nicolasdevienne** [comment]
Breaking News! Twilio has resumed development of the React Native SDK: https://github.com/twilio/react-native-twilio-video-webrtc/
